From: Nicolas Braud-Santoni <nicoo@debian.org>
Date: Sat, 16 Jan 2021 17:32:04 +0100
Subject: opamFileTools.lint: Fix typo & grammar in error message

Forwarded: https://github.com/ocaml/opam/pull/3955
---
 src/state/opamFileTools.ml | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/src/state/opamFileTools.ml b/src/state/opamFileTools.ml
index a228dbe..381f4ec 100644
--- a/src/state/opamFileTools.ml
+++ b/src/state/opamFileTools.ml
@@ -670,8 +670,9 @@ let lint ?check_extra_files ?(check_upstream=false) t =
             if not_corresponding = [] then None
             else
             let msg =
-              Printf.sprintf "Cheksum%s %s don't verify archive"
-                (if List.length chks = 1 then "" else "s")
+              let is_singular = function [_] -> true | _ -> false in
+              Printf.sprintf "Checksum%s match the archive: %s"
+                (if is_singular not_corresponding then " doesn't" else "s don't")
                 (OpamStd.List.to_string OpamHash.to_string not_corresponding)
             in
             Some msg)
