From: Nicolas Braud-Santoni <nicoo@debian.org>
Date: Sat, 16 Jan 2021 17:32:04 +0100
Subject: opamCommands: Fix typo & grammar in check

Forwarded: https://github.com/ocaml/opam/pull/3955
---
 src/client/opamCommands.ml | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/src/client/opamCommands.ml b/src/client/opamCommands.ml
index 508965d..73d6375 100644
--- a/src/client/opamCommands.ml
+++ b/src/client/opamCommands.ml
@@ -1462,8 +1462,9 @@ let update =
       "Do the update, then return with code 0 if there were any upstream \
        changes, 1 if there were none. Repositories or development packages \
        that failed to update are considered without changes. With \
-       $(b,--upgrade), behaves like $(b,opam upgrade --check), that is, \
-       returns 0 only if there are currently availbale updates." in
+       $(b,--upgrade), applies to the upgrade step: that is $(b,opam update \
+       --upgrade --check) behaves like $(b,opam update && opam upgrade --check), \
+       returning 0 if there are available upgrades, rather than upstream updates." in
   let update global_options jobs names repos_only dev_only all check upgrade =
     apply_global_options global_options;
     OpamStateConfig.update
